ee/tabby-webserver/src/hub/api.rs

```diff
@@ -23,6 +26,8 @@ pub trait Hub {
         limit: usize,
         offset: usize,
     ) -> SearchResponse;
+
+    async fn get_repositories() -> Result<Vec<RepositoryConfig>, String>;
```

Comment:
Let’s just release `Vec<RepositoryConfig>` for simplicity - you could handle error and return empty vec instead.